Src/CMakeLists.txt

```diff
@@ -52,7 +58,7 @@ if (LIBCZI_BUILD_CURL_BASED_STREAM)
     set(BUILD_CURL_EXE OFF CACHE BOOL "" FORCE)
     set(CMAKE_POSITION_INDEPENDENT_CODE ON CACHE BOOL "" FORCE)
     set(ENABLE_UNICODE ON CACHE BOOL "" FORCE)
-
+   
```

Comment:
Whitespace added unintenionally, I guess.

Reply:
Yes but no harm done I guess, it's always a good idea to have some spare whitespaces :-)